app/javascript/client/form/UpdateNumberInput.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { HTMLProps } from 'react';
+import styled from 'styled-components';
+
+interface UpdateNumberInputProps extends Omit<HTMLProps<HTMLInputElement>, 'as' | 'ref' | 'autoComplete'> {
+}
+
+const StyledNumberInput = styled.input`
+    width: 75%;
+`;
+
+export const UpdateNumberInput = React.forwardRef(( { ...rest }: UpdateNumberInputProps, ref) => (
```

Comment:
added this field so that a min quantity of `0` could be entered, will probably add a `remove from cart` button as well just for more options

Reply:
:+1: Sounds good, yeah I agree that'd be a nice thing to have!